api/src/index.ts

```diff
@@ -79,7 +79,3 @@ const errorHandler = (err: Error, req: Request, res: Response) => {
 
 app.use(errorHandler);
 
-app.listen(config.httpPort, async () => {
```

Comment:
`app.listen`is needed in express framework

Reply:
yeah this is a work in progress.  Had to remove it to run some tests but will add it back shortly.

Reply:
Just to give some context. I want to force the integration tests to fail hence removing the `app.listen` was a simple way to achieve this.